src/utils.ts

```diff
@@ -98,3 +98,5 @@ export function deepEqual(obj1: any, obj2: any) {
 
   return true;
 }
+
+export const maxSafeTimeout = 2 ** 31 - 1;
```

Comment:
I'd change this name to represent what it actually is (for example in C there is `INT_MAX`)

Reply:
I think `INT_MAX` overlaps with existing `MAX_SAFE_INTEGER` without providing context for why it exists.
What about `MAX_SAFE_INT32` or `MAX_SAFE_TIMEOUT`?
The former signals it's the max 32 bit value while the latter that it's safe for timeouts.

Reply:
I'd like to avoid using `timeout`. If we were to use it later, it would be confusing. `INT32_MAX` sounds ok.